sort.c

```diff
@@ -0,0 +1,109 @@
+/*
+ * This file is part of RedisRaft.
+ *
+ * Copyright (c) 2020-2021 Redis Ltd.
+ *
+ * RedisRaft is licensed under the Redis Source Available License (RSAL).
+ */
+
+#include <strings.h>
+
+#include "redisraft.h"
+
+static void replySortedArray(RedisModuleCtx *ctx, RedisModuleCallReply * reply)
+{
+    size_t len = RedisModule_CallReplyLength(reply);
+    if (len == 0) {
+        RedisModule_ReplyWithNullArray(ctx);
+        return;
+    }
+
+    RedisModuleDict * dict = RedisModule_CreateDict(ctx);
+    for(int i=0; i < len; i++) {
+        RedisModuleCallReply * entry = RedisModule_CallReplyArrayElement(reply, i);
+        size_t entry_len;
+        const char * entry_str = RedisModule_CallReplyStringPtr(entry, &entry_len);
```

Comment:
```suggestion
    for (size_t i = 0; i < len; i++) {
        RedisModuleCallReply *entry = RedisModule_CallReplyArrayElement(reply, i);
        size_t entry_len;
        const char * entry_str = RedisModule_CallReplyStringPtr(entry, &entry_len);
```

Styling/spacing and type correctness.